src/plasmapy/utils/data/downloader.py

```diff
@@ -13,14 +13,24 @@
 
 import requests
 
-__all__ = ["Downloader"]
+__all__ = ["_API_CONNECTION_ESTABLISHED", "Downloader"]
 
+_API_CONNECTION_ESTABLISHED = False
 _IS_CI = "GH_TOKEN" in os.environ
 
 
 # TODO: use a config file variable to allow users to set a location
 # for the data download folder?
 
+try:
```

Comment:
I suppose this has to be in this main file rather than in the test file since multiple different test files (anything using downloader) will need to access it?

Reply:
Yep, we just import it from there into the test modules that use the downloader directly/indirectly